lib/x11utils.pm

```diff
@@ -106,8 +106,14 @@ sub ensure_unlocked_desktop {
             record_info('Guest disabled display', 'Might be consequence of bsc#1168979');
         }
         if (match_has_tag('authentication-required-user-settings') || match_has_tag('authentication-required-modify-system')) {
-            type_password;
-            assert_and_click "authenticate";
+            wait_still_screen(2);    # Check again as the pop-up may be just a glitch, see bsc#1168979
```

Comment:
I'm not sure if it's going to work, because `authentication-required-user-settings` tag will match polkit https://openqa.suse.de/tests/4823757#step/change_password/8
Will 2 seconds be sufficient? Do we also want to limit this delay to `s390x` only?

Reply:
I'd add a `send_key 'shift';` to force a screen redraw, however as this is happening inside ensure_unlocked_desktop, we should be covered. We see it on s390 because it's the only place where the VNC connection is being detected as a remote one (???), however it could happen in other arches if VNC is remote :)

Reply:
I ignored that trick with shift, sounds like a good idea. Otherwise in all VRs so far two seconds were enough. But we could just let the default 7 seconds jsut to be sure

Reply:
> I'm not sure if it's going to work, because `authentication-required-user-settings` tag will match polkit https://openqa.suse.de/tests/4823757#step/change_password/8
> Will 2 seconds be sufficient? Do we also want to limit this delay to `s390x` only?

I don't see why it wouldn't work here, we're safe as we double check the needle. I'm not sure sending shift key will work, so I just removed the limit of two seconds to be safe. I'd rather not add another if statement here to limit it to s390 and, as foursixnine pointed out, it might happen as well on other non-kvm backends.